micronaut-camunda-bpm-feature/src/test/java/info/novatec/micronaut/camunda/bpm/feature/test/MicronautProcessEngineConfigurationTest.java

```diff
@@ -70,7 +71,7 @@ void allBeansAreAvailableInApplicationContext() {
 
     @Test
     void testDeploymentName() {
-        assertEquals(MicronautProcessEngineConfiguration.MICRONAUT_AUTO_DEPLOYMENT_NAME, repositoryService.createDeploymentQuery().singleResult().getName());
+        Assertions.assertEquals(MicronautProcessEngineConfiguration.MICRONAUT_AUTO_DEPLOYMENT_NAME, repositoryService.createDeploymentQuery().singleResult().getName());
```

Comment:
I don't think this change was intentional: please remove the reference to "Assertions". You already have the static import to assertEquals.

The import `import org.junit.jupiter.api.Assertions` is then also obsolete.

Reply:
Idea must have done it during automatic refactoring. Fixed.